src/modules/file/file.usecase.ts

```diff
@@ -309,6 +309,14 @@ export class FileUseCases {
     await this.fileRepository.deleteFilesByUser(user, files);
   }
 
+  /**
+   * Deletes files regardless of the user. Useful in the case of workspace deletion.
+   * @param files Files to be deleted
+   */
+  async delete(files: File[]): Promise<void> {
```

Comment:
Enforce the user id usage, as this operation is delicate so we do not want to use the wrong file ids and accidentally delete files that we do not want to delete